opentracing-api/src/main/java/io/opentracing/ScopeManager.java

```diff
@@ -26,15 +26,28 @@
 public interface ScopeManager {
 
     /**
+     * @deprecated use {@link #activate} instead.
      * Make a {@link Span} instance active.
      *
      * @param span the {@link Span} that should become the {@link #active()}
      * @param finishSpanOnClose whether span should automatically be finished when {@link Scope#close()} is called
      * @return a {@link Scope} instance to control the end of the active period for the {@link Span}. It is a
      * programming error to neglect to call {@link Scope#close()} on the returned instance.
      */
+    @Deprecated
     Scope activate(Span span, boolean finishSpanOnClose);
 
+    /**
+     * Make a {@link Span} instance active.
```

Comment:
add some description of what active means as the word isn't intuitive.

For example "Makes this span visible via commands like (XX) until the corresponding scope is close"

Also, note that it is a programming error to leave the scope unclosed or to attempt to close it on a different thread.

Reply:
Thanks, will add such explanation.